src/components/ebay-snackbar-dialog/component.js

```diff
@@ -0,0 +1,22 @@
+const DEFAULT_TIMEOUT_LENGTH = 6000;
+module.exports = {
+    onInput(input) {
+        if (input.open === true || input.open === false) {
+            this.state.open = input.open;
```

Comment:
Why do we need this?
It doesn't look like state is ever changed

Reply:
good catch